README.md

```diff
@@ -1,9 +1,8 @@
 # runnable-exports
-[![Build Status](https://travis-ci.org/LasaleFamine/runnable-exports.svg?branch=master)](https://travis-ci.org/LasaleFamine/runnable-exports) [![Coverage Status](https://coveralls.io/repos/github/LasaleFamine/runnable-exports/badge.svg)](https://coveralls.io/github/LasaleFamine/runnable-exports)
-
+[![Build Status](https://travis-ci.com/emurphy-9/runnable-exports.svg?branch=master)](https://travis-ci.com/emurphy-9/runnable-exports) [![Coverage Status](https://coveralls.io/repos/github/emurphy-9/runnable-exports/badge.svg?branch=master)](https://coveralls.io/github/emurphy-9/runnable-exports?branch=master)
```

Comment:
I think this should be reverted 😄

Reply:
I'll take care of this 🤙 I will close the related issue #1 when released. Thanks a lot again!

Reply:
Awesome! No problem, it was my pleasure. Thanks for taking the time to respond and merge everything in! 🙂